packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -33,6 +34,14 @@ export const RelationInputDataManger = ({
     useCMEditViewDataManager();
   const [{ query }] = useQueryParams();
 
+  const getEntityId = () => {
```

Comment:
Why does this need to be a method instead of a static value?

Reply:
I went with this first
```
entityId: isCreatingEntry ? undefined : componentUid ? get(initialData, name.split('.').slice(0, -1))?.id : initialData.id;
```
but lint doesn't appreciate nested ternary condition - do I miss another way to handle it? 👀

Reply:
fixed [here](https://github.com/strapi/strapi/pull/14544/commits/266061f430ee8aacfc0c0def93f7732ebd5dea05)